common/src/app/common/pages/changes_builder.cljc

```diff
@@ -174,15 +208,15 @@
   ([changes ids update-fn {:keys [attrs ignore-geometry?] :or {attrs nil ignore-geometry? false}}]
    (assert-page-id changes)
    (assert-objects changes)
-   (let [objects (::objects (meta changes))
+   (let [objects (-> changes meta ::file-data (get-in [:pages-index uuid/zero :objects]))
```

Comment:
The same, don't mix two stilles when you already have to use the get-in

Reply:
ok